datameta/api/metadatasets.py

```diff
@@ -150,6 +150,12 @@ def post(request: Request) -> MetaDataSetResponse:
     # Obtain string converted version of the record
     record = record_to_strings(request.openapi_validated.body["record"])
 
+    replaces = request.openapi_validated.body.get("replaces")
+    replaces_label = request.openapi_validated.body.get("replaces_label")
```

Comment:
This needs to be added to the API schema. To maintain API downstream compatibility, they must not be required.

Reply:
Yea, I thought that was already in the `additionalProperties`, but overlooked that this is set to `false` anyway.